apache/trafficcontrol
# TP: adds the ability to clone a topology

## What does this PR (Pull Request) do?

- [x] This PR fixes #5360 

![image](https://user-images.githubusercontent.com/251272/103673894-54d76880-4f3b-11eb-94e5-9fea099120fa.png)

## Which Traffic Control components are affected by this PR?

- Traffic Portal

## What is the best way to verify this PR?
- Navigate to https://tp.domain.tld/#!/topologies and select a topology
- Click `More > Clone Topology'
- Provide a new name for the clone and hit `Clone`
- Ensure a new topology was created with the same cache groups and same cache group hierarchy with the new name and is assigned to NO delivery services.

## The following criteria are ALL met by this PR
